[
{"message":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","level":"FATAL","type":"PLAYGROUND","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","log":"/tmp/tmp.N48bA82Akx"}
,
{"message":"Uncaught Error: Undefined constant \"affiche_poker\" in /wordpress/wp-content/plugins/cards-poker/Cards_poker.php:64","level":"FATAL","type":"PHP","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:31:18 UTC] PHP Fatal error:  Uncaught Error: Undefined constant \"affiche_poker\" in /wordpress/wp-content/plugins/cards-poker/Cards_poker.php:64\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('cards-poker/Car...')\n#2 /internal/eval.php(20): activate_plugin('cards-poker/Car...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/cards-poker/Cards_poker.php on line 64","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"Uncaught Error: Undefined constant \"affiche_poker\" in /wordpress/wp-content/plugins/cards-poker/Cards_poker.php:64","level":"FATAL","type":"PHP","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:31:26 UTC] PHP Fatal error:  Uncaught Error: Undefined constant \"affiche_poker\" in /wordpress/wp-content/plugins/cards-poker/Cards_poker.php:64\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('cards-poker/Car...')\n#2 /internal/eval.php(20): activate_plugin('cards-poker/Car...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/cards-poker/Cards_poker.php on line 64","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"Uncaught Error: Undefined constant \"affiche_poker\" in /wordpress/wp-content/plugins/cards-poker/Cards_poker.php:64","level":"FATAL","type":"PHP","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:31:34 UTC] PHP Fatal error:  Uncaught Error: Undefined constant \"affiche_poker\" in /wordpress/wp-content/plugins/cards-poker/Cards_poker.php:64\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('cards-poker/Car...')\n#2 /internal/eval.php(20): activate_plugin('cards-poker/Car...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/cards-poker/Cards_poker.php on line 64","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method SendinBlue_CF7::activate() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324","level":"FATAL","type":"PHP","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:33:37 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method SendinBlue_CF7::activate() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_cf7-se...', false)\n#3 /internal/eval.php(20): activate_plugin('cf7-sendinblue-...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method SendinBlue_CF7::activate() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324","level":"FATAL","type":"PHP","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:33:45 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method SendinBlue_CF7::activate() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_cf7-se...', false)\n#3 /internal/eval.php(20): activate_plugin('cf7-sendinblue-...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method SendinBlue_CF7::activate() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324","level":"FATAL","type":"PHP","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:33:53 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method SendinBlue_CF7::activate() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_cf7-se...', false)\n#3 /internal/eval.php(20): activate_plugin('cf7-sendinblue-...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method Custom_Taxonomy_Order::activate() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324","level":"FATAL","type":"PHP","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:36:16 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method Custom_Taxonomy_Order::activate() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_custom...', false)\n#3 /internal/eval.php(20): activate_plugin('custom-taxonomy...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method Custom_Taxonomy_Order::activate() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324","level":"FATAL","type":"PHP","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:36:23 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method Custom_Taxonomy_Order::activate() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_custom...', false)\n#3 /internal/eval.php(20): activate_plugin('custom-taxonomy...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method Custom_Taxonomy_Order::activate() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324","level":"FATAL","type":"PHP","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:36:30 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method Custom_Taxonomy_Order::activate() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_custom...', false)\n#3 /internal/eval.php(20): activate_plugin('custom-taxonomy...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_revision_control.","level":"FATAL","type":"SQL","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:37:43 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_revision_control WHERE setting = 'revisions'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_revision_control WHERE setting = :param0  | parameters: revisions</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_revision_control.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/companion-revision-manager/companion-revision-manger.php(327): wpdb->get_results('SELECT * FROM w...')\n#3 /wordpress/wp-content/plugins/companion-revision-manager/companion-revision-manger.php(340): crm_settings()\n#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('companion-revis...')\n#6 /internal/eval.php(20): activate_plugin('companion-revis...')\n#7 {main}</pre>\n for query SELECT * FROM wp_revision_control WHERE setting = 'revisions' made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/companion-revision-manager/companion-revision-manger.php'), crm_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_revision_control.","level":"FATAL","type":"SQL","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:37:52 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_revision_control WHERE setting = 'revisions'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_revision_control WHERE setting = :param0  | parameters: revisions</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_revision_control.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/companion-revision-manager/companion-revision-manger.php(327): wpdb->get_results('SELECT * FROM w...')\n#3 /wordpress/wp-content/plugins/companion-revision-manager/companion-revision-manger.php(340): crm_settings()\n#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('companion-revis...')\n#6 /internal/eval.php(20): activate_plugin('companion-revis...')\n#7 {main}</pre>\n for query SELECT * FROM wp_revision_control WHERE setting = 'revisions' made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/companion-revision-manager/companion-revision-manger.php'), crm_settings, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>cookie-panda</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121","level":"NOTICE","type":"PHP","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:38:56 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>cookie-panda</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>cookie-panda</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121","level":"NOTICE","type":"PHP","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:39:05 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>cookie-panda</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/c/countdown-2-date/error.json"}
,
{"message":"Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>cookie-panda</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121","level":"NOTICE","type":"PHP","test":"ast-sqlite-boot","plugin":"countdown-2-date","details":"[30-May-2025 15:39:13 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>cookie-panda</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #5 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"countdown-2-date\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/countdown-2-date.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/countdown-2-date.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/countdown-2-date.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/countdown-2-date.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n","log":"/logs/plugins/c/countdown-2-date/error.json"}
]
